packages/components/tests/dummy/app/styles/components/dummy-component-props.scss

```diff
@@ -49,15 +49,28 @@
         background-color: #e4e4e4;
       }
     }
+  }
 
-    .default {
-      display: inline-block;
-      padding: 0.125rem 0.25rem;
-      border-radius: 4px;
-      font-weight: 700;
-      color: #0c56e9;
-      background-color: #cce3fe;
-    }
+  // Indicators / Tags
+  .default,
+  .required {
+    border-radius: 4px;
+    font-weight: 700;
+    display: inline-block;
+    padding: 0.125rem 0.25rem;
+  }
+
+  .default {
+    color: #0c56e9;
+    background-color: #cce3fe;
+  }
+
+  .required {
+    background-color: pink;
+    background-color: var(--token-color-surface-critical);
+    border: 1px solid var(--token-color-border-critical);
+    color: var(--token-color-foreground-critical-on-surface);
```

Comment:
we don't use design tokens (and components) from the HDS design system to build the UI of the documentation, to avoid coupling the two and making things harder for us.
it's OK if you just want to use the corresponding hex color values, though.